rspack.config.ts

```diff
@@ -105,25 +96,6 @@ const sentryDjangoAppPath = path.join(__dirname, 'src/sentry/static/sentry');
 const distPath = env.SENTRY_STATIC_DIST_PATH || path.join(sentryDjangoAppPath, 'dist');
 const staticPrefix = path.join(__dirname, 'static');
 
-// Locale file extraction build step
-if (env.SENTRY_EXTRACT_TRANSLATIONS === '1') {
-  babelConfig.plugins?.push([
-    'module:babel-gettext-extractor',
```

Comment:
if translation with babel plugin is required, you can stiil use babel plugin but it will cause large performance degradation, if theres SWC solution or you can limits the scope of the files transformed, it'll contain the blast radius

Reply:
I'm trying to get rid of babel due to the performance. Hopefully swc works correctly with emotion.